developments_build/sql/qaqc/qaqc_units.sql

```diff
@@ -37,6 +37,7 @@ JOBNUMBER_top_alt_inc AS(
     FROM UNITS_devdb
     WHERE
     job_type = 'Alteration'
+    AND classa_net IS NOT NULL
     ORDER BY classa_net DESC
     LIMIT 20
```

Comment:
should we remove the limit?

Reply:
The purpose of this check is to identify the top 20 records by increase in units.